### PR TITLE
Dtmn git training

### PR DESCRIPTION
**Pull request by DevikaNair90**

### CODE REVIEW

**Comment by DevikaNair90** [comment]
@Nosferican want to merge? I want to test running the document from rivanna

**Comment by Nosferican** [comment]
You can test it before merging (`git checkout`) but sure. Go ahead.

**Comment by Nosferican** [comment]
Looks good.

File: 03_codemanagement/git_learnr/Intro to Git.Rmd [hunk 1]
```diff
@@ -0,0 +1,612 @@
...
+    E.  Delete your branch: `git branch -d my_awesome_task` (note it is a lower case d) 
+    F.  Clean up your branches: `git fetch --prune`
+
+### Additional Learning Resources
+
+
+I highly recommend the following resources for additional learning and practice.
+
+* Datacamp https://www.datacamp.com/courses/introduction-to-git-for-data-science
+* GitHub Classroom https://classroom.github.com/
```
Comment (Nosferican) [inline 03_codemanagement/git_learnr/Intro to Git.Rmd:610]: If not mistaken the GitHub Classroom is for teaching classes using GitHub not actual Git training.
I would include rather:
- https://education.github.com/students/experts (Training in Git for students)

**Pull request merged**